linux-anvil/Dockerfile

```diff
@@ -46,6 +47,20 @@ RUN curl -s -L -O https://github.com/krallin/tini/releases/download/v0.10.0/tini
     chmod +x tini && \
     mv tini /usr/local/bin
 
+# Install gosu to be used in the entrypoint, reason why to use it instead of
+# su or sudo can be found on the project docs
+RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
+RUN curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.9/gosu-amd64" && \
+    openssl sha256 /usr/local/bin/gosu | grep 5ec5d23079e94aea5f7ed92ee8a1a34bbf64c2d4053dadf383992908a2f9dc8a && \
+    curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.9/gosu-amd64.asc" && \
+    gpg --verify /usr/local/bin/gosu.asc && \
+    rm /usr/local/bin/gosu.asc && \
+    chmod +x /usr/local/bin/gosu
+
+# give sudo permission for conda user to run yum (user creation is postponed
+# to the entrypoint, so we can create a user with the same id as the host)
+RUN echo 'conda ALL=NOPASSWD: /usr/bin/yum' >> /etc/sudoers
```

Comment:
I would prefer to limit ```sudo``` usage, but I probably don't see the whole picture here. So if it seems necessary I'm ok with that.

Just a reminder that it is always easier to add functionality than to remove it. If we restrict ```sudo``` to yum only and then later decide to give permission to any command it will be a very simple non-breaking change. The opposite would have to be much more discussed as this PR.